examples/taproot.rs

```diff
@@ -0,0 +1,44 @@
+extern crate bitcoin;
+extern crate miniscript;
+
+use std::str::FromStr;
+
+#[cfg(feature = "compiler")]
+use miniscript::policy::Concrete;
+use miniscript::Descriptor;
+
+fn main() {
+    let policies_str = [
+        "or(1@pk(A),1@pk(B))",
```

Comment:
We should make this example as a tutorial. 

- Take a example policy,
 - explain what does
 - explain the compiled result. 
 
 Do this a couple of times for interesting results. Using unspendable internal key, useful internal key being inferred etc..